src/OrchardCore.Modules/OrchardCore.ContentTypes/Views/Admin/Edit.cshtml

```diff
@@ -16,10 +17,13 @@
         .OrderBy(x => x.Order)
         .Select(x => x.Part);
 
-    var orderedFields = typePart == null ? Enumerable.Empty<ContentPartFieldDefinition>() : typePart.PartDefinition.Fields
+    var orderedFields = ((typePart?.PartDefinition?.Fields) ?? Enumerable.Empty<ContentPartFieldDefinition>())
```

Comment:
Why you didn't assign `Enumerable.Empty<ContentPartFieldDefinition>()` to `orderedFields` if the fields is `null`